e2e/stack_test.go

```diff
@@ -117,6 +122,26 @@ func TestE2E(t *testing.T) {
 	runningTests.Wait()
 }
 
+func checkForRollbacks(t *testing.T, stack *e2e.StackConfig) {
+	lastBlock, err := stack.MonomerClient.BlockByNumber(stack.Ctx, nil)
+	require.NoError(t, err)
+	for {
+		currentBlock, err := stack.MonomerClient.BlockByNumber(stack.Ctx, nil)
+		require.NoError(t, err)
+		// Ensure that the current monomer block height isn't less than the last checked monomer block height.
+		if currentBlock.Number().Int64() < lastBlock.Number().Int64() {
+			require.Fail(t, "monomer has rolled back")
+		}
+		// End the test once the target monomer block height is reached
+		if currentBlock.Number().Int64() >= int64(15) {
```

Comment:
question for reviewers - With a SWS of 4, L1 block time of 2 ,and L2 block time of 1, the rollbacks will always occur at monomer block 13 so this check is sufficient. However, if any config param ever changes, the target monomer block height may need to be tuned to correctly check for rollbacks. Should I make calculating the target block height more formulaic based on those config params?

Reply:
We should hit the rollback after a single sequencing window has passed on L1, correct? Therefore, it makes sense to wait one SWS + 1 L1 blocks.